src/cmds/init.py

```diff
@@ -377,9 +376,28 @@ def chia_init(root_path: Path):
 
     manifest = MANIFEST
 
-    # Starting with RC2 is the first version that used the bech32m addresses
-    # range current version and 0=version 1
-    # for version_number in range(chia_minor_release_number() - 1, 2, -1):
+    if root_path.is_dir() and Path(root_path / "config" / "config.yaml").exists():
+        # This is reached if CHIA_ROOT is set, or if user has run chia init twice
+        # before a new update.
+        config: Dict = load_config(root_path, "config.yaml")
+        if config["selected_network"] != "testnet8":
+            # Rc6, Rc7 -> Rc8 migration
+            tmp_path = Path(os.path.expanduser(Path("~/.chia/testnet8-tmp")))
+            if tmp_path.exists():
+                shutil.rmtree(tmp_path)
+            assert migrate_from(root_path, tmp_path, manifest, DO_NOT_MIGRATE_SETTINGS)
+            check_keys(tmp_path)
+            shutil.rmtree(root_path)
+            tmp_path.rename(root_path)
+            return 0
+        else:
+            check_keys(root_path)
+            print(f"{root_path} already exists, no migration action taken")
+            return -1
+
+    print(f"Checking {os.path.expanduser(Path('~/.chia/testnet'))}")
+
+    # Migrate only from rc2 and up. Target addresses are not migrated.
     for version_number in range(5, 2, -1):
```

Comment:
Should we just drop this?

Reply:
People are sitll migrating their plot directories and stuff, and many people are probably not farming and waiting for mainnet, so we want to migrate them too